lab-khalid/server.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const
+  express = require('express'),
+
+  morgan = require('morgan'),
+  cors = require('cors'),
+  debug = require('debug'),
+  errors = require('./lib/error-middleware.js'),
+  mongoose = require('mongoose'),
+  dotenv = require('dotenv');
+
```

Comment:
make sure to require in your authRouter file:
`authRouter = require('./route/auth-router.js');`